ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -69,6 +69,12 @@ tests:
   commands: TEST_SUITE=openshift/conformance run-tests
   openshift_ansible_40:
     cluster_profile: aws-centos-40
+- as: e2e-etcd-quorum-loss
```

Comment:
If you drop this, you'll be able to land this PR with only installer-team approval.  Then we will have some in-the-wild examples to show the MCO folks when we add this job for them.  If you keep it here, you'll need to track down an MCO approver to sign off on this PR.

Reply:
>If you drop this, you'll be able to land this PR with only installer-team approval. Then we will have some in-the-wild examples to show the MCO folks when we add this job for them

The whole idea of this PR is adding a new test for MCO, debug MCO PRs (e.g. https://github.com/openshift/machine-config-operator/pull/788) and once its ready make this test run for all MCO/MAO PRs